subgraphs/uniswap-forks/protocols/sushiswap/src/common/constants.ts

```diff
@@ -0,0 +1,8 @@
+////////////////////
+//////Versions//////
+////////////////////
+
+export const PROTOCOL_SUBGRAPH_VERSION = "1.1.1";
+export const PROTOCOL_METHODOLOGY_VERSION = "1.0.0";
+export const PROTOCOL_NAME = "Sushiswap";
+export const PROTOCOL_SLUG = "sushiswap";
```

Comment:
Generally it's a good practice to have a new line at the end of every file: https://stackoverflow.com/q/729692/1438474

Reply:
I see. Added a new line!

Reply:
Not just this file. All files

Reply:
Done